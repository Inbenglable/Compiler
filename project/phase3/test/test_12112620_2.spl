int foo(int a, float b, char c){
    a = 1;
    b = 1.111;
    c = '1';
    return a;
}
int main(){
    foo(1,1.0,'1');
    foo(1,1.0);
    foo(1,f,'1');
    return 0;
}