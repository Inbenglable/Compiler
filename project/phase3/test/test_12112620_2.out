Error type 9 at line 9: invalid argument number
Error type 1 at line 10: "f" is used without a definition
